.github/workflows/main_lexiquestapi.yml

```diff
@@ -63,7 +62,7 @@ jobs:
         id: deploy-to-webapp

```

Comment:
You might be missing some information in here, the deployment to azure was challenging for sure